docs/source/developer/architecture.rst

```diff
@@ -446,7 +446,11 @@ then be fused together into one parfor that takes only one pass over the
 data and will thus typically have better cache performance.  Finally,
 during lowering, these parfor operators are converted to a form similar
 to guvectorize to implement the actual parallelism.  All these processes
-are described in more detail in the following paragraphs.
+are described in more detail in the following paragraphs, and many of
+them can be individually controlled by setting the ``parallel`` option
+to a dictionary of sub-options instead of a boolean. The default
```

Comment:
Has the list of valid sub-options been erased in a refactor of the docs?

Reply:
I have not found a suitable place to put it. Any suggestion?

Reply:
How about here: https://github.com/numba/numba/pull/2548/files/32d87448e9baa73b00b4e51b5a45df63ce4d306a#diff-6d0d92d9e2d016833d7c67dae3572537R454 perhaps with a lead in like:
`The automatic parallelization pass has a number of sub-passes that are controllable using a dictionary of options passed via the parallel keyword argument to jit.`
<May be insert the code snippet with options?!>
Then the following description of passes can refer to how to switch them on and off perhaps?

Other suggestions are welcome, but I do feel it ought to be spelled somewhere :)

Reply:
Thanks. I added the list of options back.

Reply:
Great, thanks. Does `prange` also need to go into that list?

Reply:
Good catch, just added it. Thanks!